packages/cosmo/package.py

```diff
@@ -11,19 +11,23 @@ class Cosmo(MakefilePackage):
     """COSMO: Numerical Weather Prediction Model. Needs access to private GitHub."""
 
     homepage = "http://www.cosmo-model.org"
-    url      = "cosmo"
-
+    url      = "https://github.com/COSMO-ORG/cosmo/archive/5.06.tar.gz" 
     git      = 'git@github.com:COSMO-ORG/cosmo.git'
     maintainers = ['elsagermann']
 
     version('master', branch='master')
     version('mch', git='git@github.com:MeteoSwiss-APN/cosmo.git', branch='mch')
-    version('5.05a', commit='ef85dacc25cbadec42b0a7b77633c4cfe2aa9fb9')
-    version('5.05',  commit='5ade2c96776db00ea945ef18bfeacbeb7835277a')
-    version('5.06', commit='26b63054d3e98dc3fa8b7077b28cf24e10bec702')
+    version('5.07.mch1.0.p4', git='git@github.com:MeteoSwiss-APN/cosmo.git', tag='5.07.mch1.0.p4')
+    version('5.07.mch1.0.p3', git='git@github.com:MeteoSwiss-APN/cosmo.git', tag='5.07.mch1.0.p3')
+    version('5.07.mch1.0.p2', git='git@github.com:MeteoSwiss-APN/cosmo.git', tag='5.07.mch1.0.p2')
+    version('5.05a', tag='5.05a')
+    version('5.05',  tag='5.05')
+    version('5.06', tag='5.06')
     
-    depends_on('netcdf-fortran')
-    depends_on('netcdf-c')
+    patch('patches/5.07.mch1.0.p4/patch.Makefile', when='@5.07.mch1.0.p4')
+
+    depends_on('netcdf-fortran~mpi')
```

Comment:
I think we should depend on netcdf w/o mpi, isnt ? However I dont know why the concretizer is failing here, saying that the spec requires +mpi